src/api/server.ts

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import { RifmDAO } from '../data/rifmDAO';
+import { SpotifyDAO } from '../data/spotifyDAO';
+
+export class APIServer {
+    private port: string;
+    private spotify: SpotifyDAO;
+    private rifm: RifmDAO;
+
+    constructor(port: string, spotify: SpotifyDAO, rifm: RifmDAO) {
```

Comment:
Fun optional ts tidbit: You can declare all of this like:
```
constructor(private spotify: SpotifyDAO, private rifm: RifmDAO, private port: string = '3000') {}
```

Reply:
And not need the members declared above

Reply:
🤯